etcdctl/ctlv2/ctl.go

```diff
@@ -49,6 +51,9 @@ func Start() {
 		cli.StringFlag{Name: "username, u", Value: "", Usage: "provide username[:password] and prompt if password is not supplied."},
 		cli.DurationFlag{Name: "timeout", Value: 2 * time.Second, Usage: "connection timeout per request"},
 		cli.DurationFlag{Name: "total-timeout", Value: 5 * time.Second, Usage: "timeout for the command execution (except watch)"},
+		// for coverage testing
```

Comment:
is it necessary to have these even though the flags in `os.Args` will be ignored?